README.md

```diff
@@ -321,12 +321,15 @@ The majority of the LLM backbones are trained on a very similar corpus of data.
 
 To train a chatbot style model, you need to convert your data into a question and answer format.
 
-> ❓ I encounter GPU out-of-memory issues. What can I change to be able to train large models? 
+If you really want to continue pretraining on your own data without teaching a question answering style, prepare a dataset with all your data in a single column Dataframe. Make sure that the length of the text in each row is not too long. In the experiment setup, remove all additional tokens (e.g. `<|prompt|>`, `<|answer|>`, for `Text Prompt Start` and `Text Answer Start` respectively) and disable `Add Eos Token To Prompt` and `Add Eos Token To Answer`. Deselect everything in the `Prompt Column`.

```

Comment:
Does this work well with LORA or does it need non-LORA?

Reply:
Also shall we outsource this to maybe WIKI?

Reply:
Would an empty string prompt column also work?

Reply:
> Also shall we outsource this to maybe WIKI?

I think it will be great in the docs (i see we have a FAQ section in the documentation PR)

Reply:
> Does this work well with LORA or does it need non-LORA?

It does work with LORA. I am not very sure about the quality, as this is not well tested and we have no benchmark. Will be happy to take suggestions w.r.t. automated tests/benchmarks on this.